lib/std/hash_map.zig

```diff
@@ -1,3 +1,6 @@
+//! Hash map library for data structures that map keys to values.
+//! If you're just looking for a map data structure, start with
+//! `StringHashMap` or `AutoHashMap`.
```

Comment:
:x: repeats the word "hash" in definition of thing with "hash" in the name
:x: "data structures" ? no, it's only 1 data structure in this file
:x: the word "just" should never be used in technical writing
:x: tells the user what to do rather than informing the user of the situation
:x: missing explanation for why to look at StringHashMap or AutoHashMap
:x: fails to mention array hash maps